cookbooks/cdo-apps/templates/default/unicorn.sh.erb

```diff
@@ -15,6 +15,8 @@ PID=<%= @pid_file %>
 CMD="cd $APP_ROOT; LANG=en_US.UTF-8 bundle exec unicorn -D -c <%= @src_file %> -E <%= @env %>"
 AS_USER=<%= @user %>
 <%= @bundle_env ? @bundle_env.map{|k,v|"export #{k}=#{v}"}.join("\n") : '' %>
+UPGRADE_DELAY=${UPGRADE_DELAY-4}
```

Comment:
What is this line doing? I see it is of the form of line 12 so intentional but I don't understand the magic constant...